charts/inbox-agent/templates/webhook/deployment.yaml

```diff
@@ -73,12 +73,12 @@ spec:
           valueFrom:
             secretKeyRef:
               name: {{ include "inbox-agent.fullname" . }}-config
-              key: endpoint
-        - name: INBOX_SERVER_TOKEN
```

Comment:
we don't need this for webhook.

Reply:
The webhook fails to detect the endpoint if they aren't set in both places for some reason...